Makefile

```diff
@@ -85,6 +85,7 @@ CARGO_TARGET_DIR ?= $(CURDIR)/target
 # Build-time environment, captured for reporting by the application binary
 BUILD_INFO_GIT_FALLBACK := "Unknown (no git or not git repo)"
 BUILD_INFO_RUSTC_FALLBACK := "Unknown"
+export TIKV_BUILD_VERSION = $(shell awk -F ' = ' '$$1 ~ /version/ { gsub(/[\"]/, "", $$2); printf("%s",$$2) }' Cargo.toml)
```

Comment:
why we need this TIKV_BUILD_VERSION

Reply:
Seems we don't xD

Reply:
Seems on Master the `git describe --tag` gives us 3.0.0-beta which is not right. :(